src/SFA.DAS.LevyTransferMatching.Web/Models/Opportunities/ApplicationDetailsViewModel.cs

```diff
@@ -20,5 +20,6 @@ public class ApplicationDetailsViewModel : ApplyRequest
         public bool? HasTrainingProvider { get; set; }
         public SelectStandardViewModel SelectStandardViewModel { get; set; }
         public int SelectedStandardId { get; set; }
+        public string CurrentFinancialYear { get; set; }
```

Comment:
I was going to ask whether this should be a datetime data type, but I see that it originates from the tax year description string.